app/routes/index.tsx

```diff
@@ -112,14 +123,17 @@ export default function App() {
         ))}
       </main>
       <footer>
-        <a href="https://stampy.ai/wiki/Stampy">About</a>
-        <a href="https://stampy.ai/wiki/Get_involved">Get Involved</a>
+        <a href="https://discord.gg/vjFSCDyMCy">Join Discord</a>
+        <a href="https://all.aisafety.info/">All answers</a>
+        <a href="https://get_involved.aisafety.info">About / Get Involved</a>
+        <a href="https://dashboard.aisafety.info">Dashboard</a>
         <a href="https://github.com/StampyAI/stampy-ui">Help Code</a>
-        <a href="https://stampy.ai/wiki/Discord_invite">Join Discord</a>
         <a href="https://docs.google.com/forms/d/e/1FAIpQLSdT--8lx5F2pAZoRPPkDusA7vUTvKTVnNiAb9U5cqnohDhzHA/viewform">
           Feedback
         </a>
-        <a href="https://stampy.ai/wiki/Meta:Copyrights">@ 2022 stampy.ai</a>
+        <a href="https://coda.io/d/AI-Safety-Info-Dashboard_dfau7sl2hmG/Copyright_su79L#_luPMa">
+          @ 2022 stampy.ai
```

Comment:
this should probably be changed to some kind of auto incrementor or something :D

Reply:
hmm, and we should also use copyright symbol © instead of @